src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  &__title {
+    padding-inline: 16px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+
+    position: absolute;
+    top: 206px;
```

Comment:
Consider using relative units (like percentages or ems) for positioning to improve responsiveness, especially for the `top` property.